### PR TITLE
Homework: calculate

### PR DESCRIPTION
**Pull request by mariuszlisowski**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @mariuszlisowski

**Comment by ziobron** [comment]
⏱ 3 XP extra for @mariuszlisowski for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+        if (second) {
+            result = first / second;
+        } else {
+            return "Division by 0";
+        }
+    } else {
+        return "Invalid data";
+    }
+    
+    return std::to_string(result);
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:23]: You can also immediately return values. But this is my preference. Both solutions are good.

**Review by nauka-programowania-MA — APPROVED** [review]
Perfect! I really enjoy your solutions!

**Pull request closed without merging**